### PR TITLE
Override viewport when desktop mode enabled

### PR DESCRIPTION
**Pull request by cmonfortep**
Task/Issue URL: https://app.asana.com/0/414730916066338/1172939626954562/f

### Description
To improve our desktop mode on mobile, we are overriding viewport width when desktop mode is enabled

### Steps to test this PR

_Desktop site_
- [ ] go to stackoverflow
- [ ] change to desktop site (overflow menu)
- [ ] ensure site loads desktop site instead of mobile site

(trying the same with the current production app will load mobile site always)

### UI changes
| Before  | After |
| ------ | ----- |
old-desktop-mode|new-desktop-mode

### CODE REVIEW

File: app/src/main/java/com/duckduckgo/app/browser/defaultbrowsing/ViewportMod.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+class DesktopViewportMod @Inject constructor() : ViewportMod {
+    override fun onPageStarted(
+        webView: WebView,
+        url: String?,
+        desktopMode: Boolean
+    ) {
+        if (desktopMode) {
+            webView.loadUrl(
+                "javascript:document.getElementsByName('viewport')[0].setAttribute('content', " +
+                    "'width=980px');"
```
Comment (cmonfortep) [inline app/src/main/java/com/duckduckgo/app/browser/defaultbrowsing/ViewportMod.kt:42]: This value is based on the mozilla implementation https://hg.mozilla.org/mozilla-central/file/tip/mobile/android/geckoview/src/main/java/org/mozilla/geckoview/GeckoSessionSettings.java#l213

**Comment by catehstn** [comment]
@cmonfortep is this ready for review?

**Comment by malmstein** [comment]
@cmonfortep do you need help with this?

**Comment by malmstein** [comment]
Closing outdated PR.

**Pull request closed without merging**